### PR TITLE
Attribute pattern

### PR DESCRIPTION
**Pull request by kenwalger**
Adding the attribute pattern post to the Dev Hub and created a Schema Design Patterns series.

### CODE REVIEW

File: source/how-to/attribute-pattern.txt [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+.. ---------------------------------------------------------------
+
+.. meta-description::
+
+   Over the course of this blog post series, we'll take a look at twelve common Schema Design Patterns that work well in MongoDB. We hope this series will establish a common methodology and vocabulary you can use when designing schemas. Leveraging these patterns allows for the use of "building blocks" in schema planning, resulting in more methodology being used than art.
+
+.. twitter::
+   :site: @mongodb
+   :creator: @kenwalger
+   :title: Building with Patterns: The Attribute Pattern
```
Comment (MaBeuLux88) [inline source/how-to/attribute-pattern.txt]: Missing picture in twitter.

File: source/how-to/attribute-pattern.txt [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+.. ---------------------------------------------------------------
+
+=============================================
+Building with Patterns: The Attribute Pattern
+=============================================
+
+.. introduction::
+
+   Welcome back to the Building with Patterns series. Last time we looked
+   at the :doc:`Polymorphic Pattern </how-to/polymorphic-pattern>`
```
Comment (MaBeuLux88) [inline source/how-to/attribute-pattern.txt]: What is this :doc:? First time I see this??

Comment (MaBeuLux88) [reply source/how-to/attribute-pattern.txt]: Well I can guess what it is ;-). Is it documented somewhere?

Comment (kenwalger) [reply source/how-to/attribute-pattern.txt]: Yes. It's for an internal link.

File: source/how-to/attribute-pattern.txt (hunk 1 above)
Comment (kenwalger) [reply source/how-to/attribute-pattern.txt]: Currently don't have a social image for this series nor are we planning on promoting it in the near future.

**Pull request merged**